### ISSUE TITLE
[KEYBOARD] VA.gov Direct Deposit - Tab and focus order is not correct

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#2808]
## Description

[KEYBOARD] VA.gov Direct Deposit - Tab and focus order is not correct

1. **Misplaced Focus** – The focus goes behind the modal box in “edit your direct deposit information” (shift tab in the modal box).
![image](https://user-images.githubusercontent.com/57469/67704214-0fa8c580-f98b-11e9-909f-d2c0ce00a751.png)

2. Upon selecting account type the modal box appears when entering in information the submit/Ok button is greyed out and is spinning.  For users the tab order then never goes to the cancel or ok button and skips right to the close the window button:
![image](https://user-images.githubusercontent.com/57469/67704274-294a0d00-f98b-11e9-9cbd-366429578a4a.png)

## Point of Contact

Jennifer

## Acceptance Criteria

As a keyboard user, I want to open the direct deposit interface and be able to tab through the form, and the focus remain in the modal I'm interacting with while the modal is displayed.

## Steps to Recreate

1. Log into staging.va.gov with user 150 
2. Click on the person menu in the top right corner
3. Click "Profile" from that menu. Your profile view will load.
4. Click the "Direct deposit information" link to jump to the direct deposit section
5. Click any of the three buttons styled as links underneath Bank name, Account number, or Account type to open the DD modal.

**Comment by bradl3yC** [comment]
@jenstrickland with the recent changes to hiding DD modal unless DD has been configured, only user 1 can get into the DD modal.  Also, this may be isolated to IE only as the bug doesn't seem to be reproducible in chrome.  Can we confirm that isolation?

**Comment by jenstrickland** [comment]
@bradl3yC & @Justin-Pickett — I'm trying to test this with user 150, and then I tried another half dozen, but am unable to find a working direct deposit profile. Is there a test profile you're using reliably?

**Comment by bradl3yC** [comment]
@jenstrickland use user 1.  It is the only user with DD configured.

**Comment by Justin-Pickett** [comment]
@bradl3yC and @jenstrickland I am going to close out ticket # [2809](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/2809)  as it appears be a duplicate of ticket this ticket.

**Comment by jenstrickland** [comment]
@bradl3yC @Justin-Pickett @1Copenut — this appears to be resolved. I've tested in Chrome, Edge, Firefox, IE, and Safari.

**Comment by jenstrickland** [comment]
@Justin-Pickett - #2809 wasn't a duplicate, although it looks like the solution for one solved the problem for the other. @1Copenut had the tab order and focus issues as separate issues, so we created separate tickets.
